target/ppc/int_helper.c

```diff
@@ -910,6 +910,79 @@ VCT(uxs, cvtsduw, u32)
 VCT(sxs, cvtsdsw, s32)
 #undef VCT
 
+#define XVIGER_NAME(bits, rank)                                         \
+                    glue(glue(helper_xvi, bits), glue(ger, rank))
+
+#define VsrN(i) VsrB((i >> 1)) >> ((i+1) % 2)*4 & 0xF
```

Comment:
`VsrN` was a bad idea in the end. Because of the differences with `xvi8`, I suggested the changes in https://github.com/PPC64/qemu/pull/26/commits/ec8c278d293e4de2930e8cbb099510f9649599f2, and now we can drop `VsrN` and access `VsrB`/`VsrSB` directly in the `GER` macro. We still need a better name for this macro though, [GER](https://spec.oneapi.io/versions/latest/elements/oneMKL/source/domains/blas/ger.html) is the name of the entire operation.